lib/Capnp/Message.hs

```diff
@@ -445,11 +449,11 @@ write MutSegment{vec} (WordCount i) val = do
     SMV.write vec i (toLE64 val)
 
 -- | @'newSegment' msg sizeHint@ allocates a new, initially empty segment in
--- @msg@ with a capacity of @sizeHint@. It returns the a pair of the segment
+-- @msg@ with a capacity of @sizeHint@ words. It returns the a pair of the segment
```

Comment:
This suggests another improvement: probably `sizeHint` should be of type `WordCount` as well.

Reply:
I've incorporated this change and my suggestion here into master.